data-exporter/src/main/java/org/reactome/release/dataexport/UniProtReactomeEntry.java

```diff
@@ -267,14 +336,15 @@ private void setDbId(long dbId) {
 	 * a 6 or 10 character String)
 	 */
 	private void setAccession(String accession) {
-		final List<Integer> ACCEPTED_UNIPROT_ACCESSION_LENGTHS = Arrays.asList(6, 10);
 		if (accession == null) {
 			throw new NullPointerException("UniProt Accession is null");
 		}
 
-		if (!ACCEPTED_UNIPROT_ACCESSION_LENGTHS.contains(accession.length())) {
+		if (!isValidCanonicalAccession(accession) && !isValidIsoformAccession(accession)) {
```

Comment:
If this was ever changed to use ReactomeJavaConstants as an imported static class, this 'access' variable will become problematic. Just some food for thought!

Reply:
@cookersjs Huh?

Reply:
In orthoinference, we have throughout the code imported ReactomeJavaConstants statically, so that things like `ReactomeJavaConstants.Pathway` can actually be written as `Pathway`. Joel has been an advocate for that in the past (though not recently), and so if anyone was to ever implement that in this code, the variable `accession` might cause complications

Reply:
Fair point - if ReactomeJavaConstants were imported statically, they could cause confusion with a local method variable shadowing it.  I can make the variable name more specific like `uniProtAccession` to avoid that.

Reply:
Which is why I'm not a fan of static imports like that. ;)

Reply:
Yea while the static imports did make the code look cleaner, I think that they did make the code a bit more esoteric. Maybe we should discuss this a bit so that we can decide if we want to have them in the release code? Right now, we've Frankensteined the codebase a bit with Orthoinference having static imports but other modules not

Reply:
It's aliiivve!

Yeah, I think the static importing suggestion I made for those variables was misguided.  Using the `ReactomeJavaConstants` prefix may be the better way to go.